website/blog/2023-04-19-running-a-local-kubernetes-cluster-with-podman-desktop.md

```diff
@@ -0,0 +1,243 @@
+
+Podman Desktop can help you run Kind-powered local Kubernetes clusters on a container engine, such as Podman.
+
+Podman Desktop helps you [installing the `kind` CLI](/docs/kubernetes/kind/installing-kind):
+
+1. In the status bar, click on **Kind**, and follow the prompts.
+1. When the `kind` CLI is available, the status bar does not display **Kind**.
+1. On Windows, [configure Podman in rootful mode](docs/kubernetes/kind/configuring-podman-for-kind-on-windows)
+
+   ```shell-session
+   $ podman system connection default podman-machine-default-root
+   ```
+
+1. Go to **<icon icon="fa-solid fa-cog" size="lg" /> Settings > Resources**
+1. In the **<img src="/img/kind-icon.png" alt="Kind icon" style={{height: '1.5em', display: 'inline'}} /> Kind** tile, click on the **Create new** button.
+   1. **Name**: enter `kind-cluster`.
+   1. **Provider Type**: select `podman`.
+   1. **HTTP Port**: select `9090`.
+   1. **HTTPS Port**: select `9443`.
+   1. **Setup an ingress controller**: `Enabled`
+   1. Click the **Create** button.
+      ![Create a Kind cluster screen](img/running-a-local-kubernetes-cluster-with-podman-desktop/create-a-kind-cluster.png)
+1. After successful creation, click on the **Go back to resources** button
+
+#### Verification
+
+1. In **<icon icon="fa-solid fa-cog" size="lg" /> Settings > Resources** your Kind cluster is running/
+
+   ![Kind cluster is running](img/running-a-local-kubernetes-cluster-with-podman-desktop/kind-cluster-is-running.png)
+
+1. In the **Podman Desktop** tray, open the **Kubernetes** menu: you can set the context to your Kind cluster: `kind-kind-cluster`.
+
+   ![Kind cluster Kubernetes context in the tray](img/running-a-local-kubernetes-cluster-with-podman-desktop/kind-cluster-context-in-the-tray.png)
+
+   At this point, you can start working with containers, and your local Kubernetes cluster.
+
+#### Additional resources
+
+- [Creating a local Kind-powered Kubernetes cluster](docs/kubernetes/kind/creating-a-kind-cluster)
+
+## Starting the Redis leader
+
+The Guestbook application uses Redis to store its data.
+
+With Podman Desktop, you can prepare the Redis leader image and container on your local container engine, and deploy the results to a Kubernetes pod and service.
+This is functionally equal to the `redis-leader` deployment that the Kubernetes example propose.
+
+#### Procedure
+
+1. Open **<icon icon="fa-solid fa-cloud" size="lg" /> Images > <icon icon="fa-solid fa-arrow-circle-down" size="lg" /> Pull an image**.
+   1. **Image to Pull**: enter `docker.io/redis:6.0.5`
+   1. Click **Pull image** to pull the image to your container engine local image registry.
+   1. Click **Done** to get back to the images list.
+1. **<icon icon="fa-solid fa-cloud" size="lg" /> Search images**: enter `redis:6.0.5` to find the image.
+1. Click **<icon icon="fa-solid fa-ellipsis-v" size="lg" /> > <icon icon="fa-solid fa-ellipsis-v" size="lg" /> Push image to Kind cluster**.
```

Comment:
This step is not required, this is only for images that are not in a registry (locally built images)